measure-builds/src/test/java/com/automattic/android/measure/BuildTimePluginTest.kt

```diff
@@ -11,37 +11,6 @@ import java.io.File
 
 @Suppress("MaximumLineLength", "MaxLineLength")
 class BuildTimePluginTest {
-
-    @Test
-    fun `given a project that attaches gradle scan id, when executing a task with configuration from cache, then send the report with attached gradle scan id`() {
-        // given
-        val runner = functionalTestRunner(
-            enable = true,
-            attachGradleScanId = true,
-            projectWithSendingScans = true
-        )
-
-        // when
-        val prepareConfigurationCache =
-            runner.withArguments("--configuration-cache", "help").build()
-
-        // then
-        assertThat(
-            prepareConfigurationCache.output
-        ).contains("Calculating task graph as no configuration cache is available for tasks")
-            .contains("Configuration cache entry stored")
-
-        // when
-        val buildUsingConfigurationCache =
-            runner.withArguments("--configuration-cache", "help", "--debug").build()
-
-        // then
-        assertThat(buildUsingConfigurationCache.output).contains("Reusing configuration cache")
-            .contains("Reporting build data to Apps Metrics...")
```

Comment:
The new reporter doesn't have the "Reporting build data to Apps Metrics..." message, so I removed this test.

Reply:
Thanks for brining back A8c reporter in form of `InternalA8cCiReporter` and reverting removal. I'll mark this comment as resolved.